### PR TITLE
Update docker packaging for dist testing

### PR DESCRIPTION
**Pull request by sk1p**
It now installs the current ipyparallel from pypi instead of the unreleased git version.

Refs #484

## Contributor Checklist:

* [x] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [N/A] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [N/A] I have added/updated documentation for all user-facing changes
* [N/A] I have added/updated test cases
* [N/A] I have included the [rebuilt production build of the client](https://libertem.github.io/LiberTEM/contributing.html?#building-the-client) (only if changes were made to the GUI)

## Reviewer Checklist:

* [ ] `/azp run libertem.libertem-data` passed

### CODE REVIEW

File: packaging/docker/update_reqs.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-pip-compile ../../setup.py -Uo requirements.txt
```
Comment (sk1p) [inline packaging/docker/update_reqs.sh:2]: This is needed to not pollute the generated requirements.txt file with indexes specified in the user's pip configuration

**Comment by uellue** [comment]
Looks good to me! We don't use it in the LiberTEM-data pipeline, right?

**Comment by sk1p** [comment]
> Looks good to me! We don't use it in the LiberTEM-data pipeline, right?

Yep (no secure way to allow `docker-compose`, i.e. docker-in-docker)

**Comment by uellue** [comment]
Thx! :-)

**Pull request merged**